client/src/helpers/qnAHelper.js

```diff
@@ -63,89 +63,143 @@ module.exports = {
     return newArr;
 
   },
+//HOLY MOLY - this function got crazy - needed to refactor because it was duplicating answers displayed after sort
```

Comment:
this is what i was talking about.  Could using API result in less time complexity?